### ISSUE TITLE
Disabling Copper & Tin

### ISSUE THREAD

**Issue by cntrik** [Snakefangox/modfest2022#13]
Reproduce with RoughlyEnoughItems-4.10.3, TechReborn-1.16-3.4.3+build.72, RebornCore-1.16-4.3.2+build.49, fabric-api-0.16.0+build.384-1.16.1, and Mechanized-1.9.5
When disabling Copper & Zinc Ores and Ingots in the config, their respective tags itself cease to function rendering them unusable for any mod and leaving around 120 errors in the output log such as JsonSyntaxException: Unknown item tag 'c:zinc_ingots'

**Comment by snakefangox** [comment]
Huh, I suspect LibCD is disabling all mods copper and zinc tags when my ingots are disabled which was not what I was going for. Let me check if it's possible to change that.

**Comment by snakefangox** [comment]
Right, I've released a beta that should fix Mechanized nuking those tags. You'll still get some errors with it though. I'll have a proper look through LibCD and see if I can get a proper fix for this working.

**Comment by ghost** [comment]
Disabling copper & tin ore causes issues with cotton resources.
It seems Mechanized tries to register c:copper_ores and c:zinc_ores, which don't exist.
With mods like Tech Reborn, a consequence of this is that the grinders will constantly process air to air.

**Comment by snakefangox** [comment]
Ah sounds like a right mess. I'll see if I can fix it when I have time in a day or two.
